agenta-web/src/pages/apps/[app_id]/endpoints/index.tsx

```diff
@@ -40,6 +40,38 @@ const useStyles = createUseStyles((theme: JSSTheme) => ({
     },
 }))
 
+export const createParams = (
+    inputParams: Parameter[] | null,
+    environmentName: string,
+    value: string | number,
+    isChatVariant: boolean | null,
```

Comment:
Can this be inferred? How special does this have to be to deserve its own argument but can't be inferred?